src/org/pentaho/agilebi/modeler/models/annotations/UpdateCalculatedMember.java

```diff
@@ -0,0 +1,435 @@
+  private boolean inline;
+
+  @MetaStoreAttribute
+  @ModelProperty( id = FORMAT_STRING_ID, name = FORMAT_STRING_NAME, order = FORMAT_STRING_ORDER )
+  private String formatString;
+
+  @MetaStoreAttribute
+  @ModelProperty( id = FORMAT_CATEGORY_ID, name = FORMAT_CATEGORY_NAME, order = FORMAT_CATEGORY_ORDER )
+  private String formatCategory;
+
+  @MetaStoreAttribute
+  @ModelProperty( id = DECIMAL_PLACES_ID, name = DECIMAL_PLACES_NAME, order = DECIMAL_PLACES_ORDER )
+  private int decimalPlaces;
+
+  @MetaStoreAttribute
+  @ModelProperty( id = MEASURE_CONTENT_ID, name = MEASURE_CONTENT_NAME, order = MEASURE_CONTENT_ORDER )
+  private String measureContent;
+
+  @MetaStoreAttribute
+  @ModelProperty( id = CALCULATE_SUBTOTALS_ID, name = CALCULATE_SUBTOTALS_NAME, order = CALCULATE_SUBTOTALS_ORDER )
+  private boolean calculateSubtotals;
+
+  @MetaStoreAttribute
+  @ModelProperty( id = CUBE_ID, name = CUBE_NAME, order = CUBE_ORDER )
+  private String cube;
+
+  @MetaStoreAttribute
+  @ModelProperty( id = CALCULATED_MEASURE_ID, name = CALCULATED_MEASURE_NAME, order = CALCULATED_MEASURE_ORDER )
+  private String sourceCalculatedMeasure;
+
+  private transient boolean pdiContext;
+
+  /**
+   * @param workspace
+   * @param metaStore
+   * @return
+   * @throws ModelerException
+   */
+  @SuppressWarnings( "unchecked" ) @Override
+  public boolean apply( ModelerWorkspace workspace, IMetaStore metaStore ) throws ModelerException {
+    throw new UnsupportedOperationException();
+  }
+
+  /**
+   * @param schema
+   * @return
+   * @throws ModelerException
+   */
+  @Override public boolean apply( Document schema ) throws ModelerException {
+    if ( schema == null ) {
+      return false;
+    }
+
+    MondrianSchemaHandler mondrianSchemaHandler = new MondrianSchemaHandler( schema );
+
+    MondrianDef.CalculatedMember calculatedMember = new MondrianDef.CalculatedMember();
+    calculatedMember.name = this.getName();
+    calculatedMember.caption = this.getCaption();
+    calculatedMember.description = this.getDescription();
+    calculatedMember.dimension = this.getDimension();
+    calculatedMember.formula = this.getFormula();
+    calculatedMember.visible = this.isHidden();
+    MondrianDef.CalculatedMemberProperty solveOrder = new MondrianDef.CalculatedMemberProperty();
+    solveOrder.name = "SOLVE_ORDER";
+    solveOrder.value = isCalculateSubtotals() ? "200" : "0";
+    calculatedMember.memberProperties = new MondrianDef.CalculatedMemberProperty[] { solveOrder };
+
+    // annotation to indicate this was created via inline modeling, should always be true
+    MondrianDef.Annotation inline = new MondrianDef.Annotation();
+    inline.name = "AnalyzerInlineCreatedInline";
+    inline.cdata = isInline() ? "true" : "false";
+
+    // annotation to store decimal places (scale)
+    MondrianDef.Annotation formatScaleAnnotation = new MondrianDef.Annotation();
+    formatScaleAnnotation.name = "formatScale";
```

Comment:
shouldn't these be the same as in create calc measure?
https://github.com/rfellows/modeler/blob/c1498e7c9e686f032308eed8ee0a4fb46ec3e3d5/src/org/pentaho/agilebi/modeler/models/annotations/CreateCalculatedMember.java#L224

```
AnnotationConstants.INLINE_ANNOTATION_FORMAT_SCALE
```